docs/declarative-customization/column-formatting.md

```diff
@@ -20,7 +20,7 @@ The following table compares column formatting and the Field Customizer.
 
 | Field type        | Column formatting          | Field Customizer  |
 | ------------- |:-------------| :-----|
-| Conditional formatting based on item values and value ranges      | Supported | Supported |
+| https://www.youtube.com/watch?v=AmC9SmCBUj4ional formatting based on item values and value ranges      | Supported | Supported |
```

Comment:
Did you mean to insert a link here?

Reply:
Definitely should not be there... even though Gordon is entertaining, but not in this context.